docs/content/preview/yugabyte-cloud/cloud-admin/_index.md

```diff
@@ -13,7 +13,11 @@ menu:
 type: indexpage
 ---
 
-Before you can add Dedicated clusters, you need to create a billing profile and add a payment method. You review your usage, manage your billing profile and payment methods, and review invoices from the **Usage & Billing** page.
+There are no set-up charges or commitments to begin using YugabyteDB Managed. At the end of the month, you are automatically charged for that month's usage.
+
+Review your usage and plan details, manage your billing information, and review invoices from the **Usage & Billing** page.
+
+For information on available plans and their features, refer to [YugabyteDB Managed Standard Price List](https://www.yugabyte.com/yugabyte-cloud-standard-price-list/).
```

Comment:
Note that the standard price list will redirect to new pricing page. update link to www.yugabyte.com/pricing